CHANGES.rst

```diff
@@ -107,6 +107,8 @@ API changes
 
 - ``astropy.wcs``
 
+  - Improved documentation of WCS.calc_footprint [#3551]
```

Comment:
@vortex-ape - this is what I meant should be included in the changelog - the fact the ``ValueError`` is now changed (hence why it belongs in the API change section) - you indeed don't need to mention the doc change.